### DISCUSSION TITLE
ease V2 projection in LVT?

### DISCUSSION THREAD

**Discussion by slinskik** [NASA-LIS/LISF#1525] [Q&A]
A previous discussion indicated that the EASE V2 projection used by the SMAP enhanced product is supported in LDT and LIS.  I was wondering if it is also supported in LVT.  Many thanks!

**Comment by karsenau** [comment]
Hi @slinskik:

Yes, the EASE V2 grid should also have support in LVT.

If you go to your top-level "lvt" code directory, and try the following command line entries:

grep -i " ease " */*F90

Note: There should be an asterisk before and after each "/" below.
grep -i " ease " */*/*F90

And specific to "v2":
grep -i " ease" */*/*F90 | grep -i v2

You should see data stream reader examples that use that grid.
